src/stylesheet-registry.js

```diff
@@ -146,6 +153,12 @@ export default class StyleSheetRegistry {
   ) {
     const cache = {}
     return function(id, css) {
+      // Sanitize SSR-ed and client side (style tags) CSS.
+      // When using optimize for speed we don't need to sanitize
+      // because we are not inserting markup in the DOM.
+      if (!this._isBrowser || !this._optimizeForSpeed) {
```

Comment:
Why not in the browser? Just curious 🙏

Reply:
!this._isBrowser == server side rendering

Reply:
indeed I can also remove the second `!this._optimizeForSpeed`. Client side code should be safe because of `document.createTextNode` (dev) and the CSS OM api not injecting DOM (prod)